sphinx-docs/screenshot.py

```diff
@@ -23,19 +23,41 @@ def _parse_focus(arg_str):
         return {'id': split_str[0], 'annotation': split_str[1]}
 
 def _parse_command(command):
```

Comment:
This is actually a good place to write some unit tests. Are you familiar with nose (https://nose.readthedocs.org/en/latest/)?